### DISCUSSION TITLE
Can we disable Table row click globally?

### DISCUSSION THREAD

**Discussion by myregistration** [filamentphp/filament#14174] [Help]
### Package

Table builder

### Package Version

v3

### How can we help you?

Is there a way to disable Table row click globally in AppServiceProvider` boot() `so I don't have to do it for every table?  I was hoping I could use the following on something like `Filament::registerTables()`, but doesn't look like that exists.

```
->recordUrl(null)
->recordAction(null)
```

**Comment by dissto** [comment]
Can't you do:

```php

Table::configureUsing(function (Table $table) {

  return $table
      ->recordUrl(null)
      ->recordAction(null);
 }
```
 
 in a ServiceProviders `boot` method?

**Reply by myregistration** [reply]
Thanks for the suggestion.  I tried that already, but it didn't work.  Seems like it should work though.

**Reply by leandrocfe** [reply]
I don’t think that configureUsing will work at this point because the ListRecords class sets the recordUrl parameter after configureUsing

https://discord.com/channels/883083792112300104/1278889426096750664/1278959623252082753
